presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/Transport.java

```diff
@@ -91,7 +91,10 @@ private static TTransport createRaw(HostAndPort address, Optional<SSLContext> ss
 
     private static TTransportException rewriteException(TTransportException e, HostAndPort address)
     {
-        return new TTransportException(e.getType(), format("%s: %s", address, e.getMessage()), e);
+        return new TTransportException(
```

Comment:
Maybe introduce a variable to avoid excessive wrapping
```java
String message = e.getMessage() != null ? format("%s: %s", address, e.getMessage()) : address.toString();
return new TTransportException(e.getType(), message, e);